kurtosis-devnet/jovian.yaml

```diff
@@ -32,21 +32,10 @@ optimism_package:
             max_cpu: 0
             min_mem: 0
             max_mem: 0
-        node1:
-          el:
-            type: op-reth
```

Comment:
We don't yet have full support from `op-reth` for Jovian. But we would want to reinstate nodes like this as soon as possible.